### PR TITLE
Lkl list

### PR DESCRIPTION
**Pull request by cw-infinite**
1.1     The user can access the Last Known Locations (list) screen by click on Last Known Locations menu item when editing an event.
1.2     The user can see the Event Title of the event that is being edited on top of the screen
1.3          The user can see the following fields on View Last Known Location screen
                ·         Hide Inactive Location - Indicator
                ·         [Add Location] – button
                ·         Last Known Location - card
                ·         Pagination
1.4          The user can see the following fields inside each Last Known Location on the list
                ·         Location Title
                ·         Country
                ·         Post
                ·         Location Status
                ·         [Location Description] - expand chevron  (Address, Point of Contact)                       
                ·         [Additional Action] – Edit
                ·         [Additional Action] – Deactivate
1.5          The system displays the default location list with “Active” location status and sort by Last Updated Date **( wireframe does not have an indicator of the active/inactive visualization.)**
1.6          The system defaults “Hide Inactive Location” to YES  **(Still working on the UI)**
1.7          The user can distinguish active or inactive last known location visually.  **(Still working on the UI)**
1.8          The user can set “Hide Inactive Location” to NO to view all last known locations (Active and Inactive) order by default Last Updated Date  **(Still working on the UI)**
1.9          The system displays the proper pagination at bottom of the Last Known Location List, each page contains “X” number of records.  (“X” is based on C1 design)  **(Still working on the UI)** 
1.10      The user can browse through additional Last Known Locations by using the pagination controls (e.g. First, Next, Last, Page Number etc.) **(Just added a placeholder pagination)**

Added dummy data that adds "eventLklDtoList" to each event item. 
**NOTE: The dummy data I added does not have proper data format **
for ex, address might have Japan but the city is Bangkok
Basically they are not consistent because I did not want spend too much time on creating perfect dummy data. Please feel free to edit them if you need to alter data elements.

### CODE REVIEW

File: src/pages/lklList.tsx [hunk 1]
```diff
@@ -0,0 +1,270 @@
...
+	evacStatusCode?: string
+	evacDepAuthDate?: Date
+	evacDepOrdDate?: Date
+	evacSummary?: string
+    lastUpdatedDateTime?: Date
+    eventLklDtoList? : LklDto[]
+}
+
+// LklDto related interfaces
+interface PersonEmailDto  {
```
Comment (cw-infinite) [inline src/pages/lklList.tsx:38]: I separated the LKLDTO into small interfaces, I wonder if this is a right approach.

File: src/pages/lklList.tsx [hunk 2]
```diff
@@ -0,0 +1,270 @@
...
+                direction="column">
+                <Flex
+                    gridColumn={{ base: "1 / -1", md: "1 / 5", lg: "1 / 8", xl: "1 / 9" }}
+                    direction="row"
+                    wrap="wrap"
+                    justify="flex-end"
+                    mb={8}>
+
+                    <Box flexGrow={1} aria-label="Last Known Locations" >
+                        {/* CSS looks right but build has a different look than what is stated */}
```
Comment (cw-infinite) [inline src/pages/lklList.tsx:168]: Wonder if H3 is correct usage for the header in this page because it does not look right on the my local run. Let me know what you see on your local run.

Comment (jsamngc) [reply src/pages/lklList.tsx:167]: The H3 is correct. The HTML exported from Axure does not correctly render the fonts we use at certain bold (i.e. 400+) font-weights. This is an issue with the export and not with the implementation.

File: src/pages/lklList.tsx (hunk 1 above)
Comment (jsamngc) [reply src/pages/lklList.tsx:38]: That's a great way to keep everything well-organized! This is exactly what you should be doing.
